__tests__/jubilation-line/jubilation-line-test.jsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { shallow, mount } from 'enzyme';
+import JubilationLine from '../../src/jubilation-line';
+import JubilationProvider from '../../src/jubilation-provider';
+
+describe('JubilationLine', () => {
+  it.skip('should expect x1, y1, x2, y2 props', () => {
+    // TODO spy on console.log to check for React warning message
```

Comment:
Typically I would trust that Reacts PropTypes are checking the way they should and don't feel the need to test them.

Further, because we are using flow in this project, we know that prop type errors for our internal development will be caught at the time of authoring. PropType checks won't run in production so our end users won't see these checks either